package.json

```diff
@@ -16,21 +16,24 @@
     "test": "cross-env NODE_CONFIG_DIR='./tests/e2e-tests/config' BABEL_ENV=commonjs mocha \"tests/e2e-tests\" --require babel-register --recursive",
     "test:grep": "cross-env NODE_CONFIG_DIR='./tests/e2e-tests/config' BABEL_ENV=commonjs mocha \"tests/e2e-tests\" --require babel-register --grep ",
     "test:single": "cross-env NODE_CONFIG_DIR='./tests/e2e-tests/config' BABEL_ENV=commonjs mocha --require babel-register",
-    "test:vis-regression": "cross-env NODE_CONFIG_DIR='./tests/screenshot-tests/config' NODE_ENV=development JEST_PUPPETEER_CONFIG='./tests/screenshot-tests/config/jest-puppeteer.config.js' jest --config='./tests/screenshot-tests/config/jest.config.js' --maxWorkers=1 --rootDir=./",
+    "test:e2e": "./tests/bin/e2e-test-integration.js",
     "makepot": "grunt makepot",
     "git:update-hooks": "rm -r .git/hooks && mkdir -p .git/hooks && node ./node_modules/husky/husky.js install"
   },
   "devDependencies": {
+    "@wordpress/e2e-test-utils": "^2.2.0",
     "autoprefixer": "9.6.1",
-    "babel": "6.23.0",
-    "babel-cli": "6.26.0",
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/register": "^7.5.5",
     "babel-eslint": "10.0.2",
-    "babel-plugin-add-module-exports": "1.0.2",
-    "babel-preset-es2015": "6.24.1",
-    "babel-preset-stage-2": "6.24.1",
+    "babel-jest": "^24.9.0",
```

Comment:
We have babel-jest and jest at line 63 of this file. Not sure what the difference is, but perhaps only one is needed?

Reply:
@kloon good catch. Thanks for a detailed review. According to [here](https://github.com/babel/babel-jest), `babel-jest` is now automatically loaded by `Jest` and fully integrated. I removed `babel-jest`. Tested without it - everything is working.